apps/morph_stats

```diff
@@ -93,12 +93,35 @@ def _get_config(config):
         return _CONFIG
 
 
-if __name__ == '__main__':
-    args = parse_args()
-    logging.basicConfig(level=(logging.WARNING,
-                               logging.INFO,
-                               logging.DEBUG)[min(args.verbose, 2)])
-
+def _get_header(results):
```

Comment:
I would prefer it if this code was tested. It can go in `neurom.apps`. This script should stay minimal.